packages/auth/lib/AuthProviders/AuthProviderPassword.ts

```diff
@@ -0,0 +1,56 @@
+import { Service, Inject } from "@fullstack-one/di";
+import { SchemaBuilder } from "@fullstack-one/schema-builder";
+import { GraphQl } from "@fullstack-one/graphql";
+import { Auth, AuthProvider } from "..";
+
+const schema = `
+extend type Mutation {
+  """
+  Creates a new password AuthFactorCreationToken for the given password.
+  """
+  createPassword(password: String!): String! @custom(resolver: "@fullstack-one/auth/PasswordProvider/createPassword")
+
+  """
+  Creates an AuthFactorProofToken for the given user und password.
+  This will never fail. When the user could not be found it will return fake-data.
+  """
+  proofPassword(userIdentifier: String!, password: String!): String! @custom(resolver: "@fullstack-one/auth/PasswordProvider/proofPassword")
+}
+`;
+
+@Service()
+export class AuthProviderPassword {
```

Comment:
I like the consolidation of AuthFactors within the auth package! Feels very smooth.